sql/src/main/java/io/crate/operation/reference/sys/node/NodeOsStatsExpression.java

```diff
@@ -38,11 +38,9 @@ public Long innerValue() {
             }
         });
         childImplementations.put(TIMESTAMP, new SimpleNodeStatsExpression<Long>() {
-            final long ts = System.currentTimeMillis();
-
             @Override
             public Long innerValue() {
-                return ts;
+                return this.row.timestamp();
```

Comment:
this is actually null if 0, pls handle it or assert that it is not 0, since it actually must never be 0